client/src/components/solution-display-widget/index.tsx

```diff
@@ -30,34 +30,44 @@ export function SolutionDisplayWidget({
   const viewProject = t('buttons.view-project');
 
   const ShowFilesSolutionForCertification = (
-    <button
-      className='project-link-button-override'
-      data-cy={dataCy}
-      onClick={showUserCode}
-    >
-      {t('certification.project.solution')}
-    </button>
+    <Button block={true} data-cy={dataCy} onClick={showUserCode}>
+      {t('buttons.view')}
+    </Button>
   );
   const ShowProjectAndGithubLinkForCertification = (
     <>
-      <a href={solution ?? ''} rel='noopener noreferrer' target='_blank'>
-        {t('certification.project.solution')}
-      </a>
-      ,{' '}
-      <a href={githubLink} rel='noopener noreferrer' target='_blank'>
-        {t('certification.project.source')}
-      </a>
+      <div className='solutions-dropdown'>
+        <DropdownButton
+          block={true}
+          bsStyle='primary'
+          className='btn-invert'
+          id={`dropdown-for-${id}`}
+          title={t('buttons.view')}
+        >
+          <MenuItem bsStyle='primary'>
+            <a href={solution ?? ''} rel='noopener noreferrer' target='_blank'>
+              {t('certification.project.solution')}
+            </a>
+          </MenuItem>
+          <MenuItem bsStyle='primary'>
+            <a href={githubLink} rel='noopener noreferrer' target='_blank'>
+              {t('certification.project.source')}
+            </a>
+          </MenuItem>
```

Comment:
`MenuItem` is an anchor element, I hope. So you don't need to add an anchor element as its child, you can move the attributes to `MenuItem`

Reply:
because menuItem doesn't have href prop in it, it has button role which we don't want as it may confuse screen reader.

Reply:
Just to summarize
I've to modify menuItem component to accept href prop too and remove anchor tag child correct?

Reply:
I don't think you need to modify menuItem because it accepts the href prop. But yes, please remove the anchor tag child 👍 

https://github.com/freeCodeCamp/freeCodeCamp/blob/4e14899e1ce3f1b6f3ef4a5deb24240066b4baef/client/src/components/solution-display-widget/index.tsx#L115-L130